src/containers/turbo-mode.jsx

```diff
@@ -0,0 +1,60 @@
+import bindAll from 'lodash.bindall';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {connect} from 'react-redux';
+
+/**
+ * Turbo Mode component passes toggleTurboMode function to its child.
+ * It also includes `turboMode` in the props passed to the children.
+ * It expects this child to be a function with the signature
+ *     function (turboMode, toggleTurboMode, props) {}
```

Comment:
This doesn't match the example, need to remove the initial `turboMode` argument.

Reply:
Thanks! Fixed